pkg/auth/service.go

```diff
@@ -64,7 +64,7 @@ type GatewayService interface {
 }
 
 type Authorizer interface {
-	// authorize user for an action
+	// Authorize user for an action
```

Comment:
Can this be godoc please?

Reply:
Updated - let me know if it makes sense now.